scripts/python/services/test_GithubService.py

```diff
@@ -147,5 +147,27 @@ def test_throws_exception_when_client_throws_exception(self, mock_github):
             ConnectionError, github_service.close_expired_issues, "test")
 
 
+@patch("github.Github.__new__")
+class TestGithubServiceCreateAnAccessRemovedIssueForUserInRepository(unittest.TestCase):
+    def test_calls_downstream_services(self, mock_github):
+        github_service = GithubService("", ORGANISATION_NAME)
+        github_service.create_an_access_removed_issue_for_user_in_repository(
+            "test_user", "test_repository")
+        github_service.client.get_repo.assert_has_calls(
+            [call('moj-analytical-services/test_repository'),
+             call().create_issue(title=USER_ACCESS_REMOVED_ISSUE_TITLE, assignee='test_user',
+                                 body='Hi there\n\nThe user test_user had Direct Member access to this repository and access via a team.\n\nAccess is now only via a team.\n\nYou may have less access it is dependant upon the teams access to the repo.\n\nIf you have any questions, please post in (#ask-operations-engineering)[https://mojdt.slack.com/archives/C01BUKJSZD4] on Slack.\n\nThis issue can be closed.')]
```

Comment:
Can this line be changed in a new PR from (#ask-operations-engineering)[https://mojdt.slack.com/archives/C01BUKJSZD4] to use [](url)

Reply:
Good spot! Completely missed that!

I'll add a ticket for it so I don't forget 🎫